src/main/java/com/google/error_analyzer/backend/LogDao.java

```diff
@@ -147,5 +146,24 @@ private HighlightBuilder addHighLighter(String field) {
         highlightBuilder.field(highlightTitle);
         return highlightBuilder;
     }
-}
 
+    //find errors in a given index
+    private SearchHits findErrors(String fileName) 
+    throws IOException {
+        BooleanQuery booleanQuery = new BooleanQuery();
```

Comment:
This code won't compile as BooleanQuery is not available.
Can we comment this part and uncomment it when BooleanQuery PR is raised.

Also, please compile the class and check for errors.
"javac" should help.

Reply:
I checked. The code compiles